xla/tests/build_defs.bzl

```diff
@@ -21,35 +17,39 @@ GPU_BACKENDS = [
 ]
 
 # The generic "gpu" backend includes the actual backends in this list.
-GPU_DEFAULT_BACKENDS = [
+NVIDIA_GPU_DEFAULT_BACKENDS = [
     "gpu_any",
     "gpu_a100",
     "gpu_h100",
 ]
 
-_DEFAULT_BACKENDS = ["cpu"] + GPU_DEFAULT_BACKENDS
+AMD_GPU_DEFAULT_BACKENDS = ["amd_gpu_any"]
+
+_NVIDIA_DEFAULT_BACKENDS = ["cpu"] + NVIDIA_GPU_DEFAULT_BACKENDS
```

Comment:
I think we should keep this as `_DEFAULT_BACKENDS` since it also contains `cpu` and it's just the "default" which happens to include the NVIDIA targets.